src/app/helpers/announcement/controller.js

```diff
@@ -318,7 +320,7 @@ class AnnouncementController {
 
       try {
         let data = await (new Promise((resolve, reject) => {
-            this.objectStoreRest.findObject(query)
+            this.objectStoreRest.findObject(query, true)
```

Comment:
Don't pass boolean directly. Should create a variable and pass that. This helps with maintainability.